health/health_log.c

```diff
@@ -498,6 +497,32 @@ inline ALARM_ENTRY* health_create_alarm_entry(
     ae->old_value_string = strdupz(format_value_and_unit(value_string, 100, ae->old_value, ae->units, -1));
     ae->new_value_string = strdupz(format_value_and_unit(value_string, 100, ae->new_value, ae->units, -1));
 
+    if (likely(info)) {
+
+        ae->info = strdupz(info);
+
+        if ( strstr (ae->info, "$this") ) {
+            char *buf=NULL;
+            buf = find_and_replace(ae->info, "$this", ae->new_value_string);
```

Comment:
`this` is `value units`, not just a value. It affects wording and makes `$units` senseless in most of the cases.

Perhaps have `$this` as a value and use some oher var for `new_value_string`? @vlvkobal

Reply:
You are right, sorry I didn't discuss it with you before.

I think it's better to change it so that $this is always just the value, and if the user wants can use $units if he wants?

Reply:
Yes, i agree, it is better to have `$this` as just a value.

Reply:
Hi Ilya,

I've made some changes to use the raw value. I should mention though, that we do lose some of the goodies that `format_value_and_unit` provides, such as e.g. pretty print of seconds (like `00:00:02 ago` instead of just `2`), some precision changes (i.e. `97.0107422` vs `97'), and sometimes the value will be null (instead of undefined - which I don't think it will reach an alarm anyway, so it shouldn't matter). It's not bad (it's actually more 'accurate'), I think it's just a matter of what we like most.